assets/tests/components/routeLadders.test.tsx

```diff
@@ -114,11 +114,9 @@ describe("RouteLadders", () => {
       jest
         .mocked(useTimepoints)
         .mockImplementationOnce(() => timepointsByRouteId)
-      jest.mocked(useActiveDetoursByRoute).mockReturnValue({
-        "1": {
-          "1": simpleDetourFactory.build({ id: 1 }),
-          "2": simpleDetourFactory.build({ id: 2 }),
-        },
+      jest.mocked(useActiveDetours).mockReturnValue({
+        "1": simpleDetourFactory.build({ id: 1 }),
+        "2": simpleDetourFactory.build({ id: 2 }),
```

Comment:
Hmm, I'm a little surprised that this is working without `routeId` specified, do you know what I'm missing/why this works without it?

Reply:
Oooo yes, and it's totally an accident. The factory successfully generated route 1 for the first one and route 2 for the second one, but if the tests were reordered and the factory generated 3 or 4, this would break. Thanks!

Reply:
I added the route id: https://github.com/mbta/skate/pull/2930/commits/af3dd74a4c4a10f0e04a09de2904d0cf7a656ecf